stl/inc/format

```diff
@@ -1176,11 +1176,11 @@ public:
     }
 
     iterator out() {
-        return _OutputIt;
+        return _STD move(_OutputIt);

```

Comment:
So we can only call out once?

Reply:
Until we put set a new iterator, yes.

Reply:
Hmm. This seems like a defect we need to fix in the Standard; ditto the _Effects_ of `advance_to`.

Reply:
I don't want to block merging this, so I've recorded an action item to submit an LWG issue.